examples/protocols/sntp/main/sntp_example_main.c

```diff
@@ -157,10 +162,33 @@ static void initialize_sntp(void)
 {
     ESP_LOGI(TAG, "Initializing SNTP");
     sntp_setoperatingmode(SNTP_OPMODE_POLL);
+
+/*
+ * If 'NTP over DHCP' is enabled, we set static ntp server address 'pool.ntp.org'
+ * as a secondary server. It will act as a fallback server in case that address
+ * provided via NTP over DHCP is not accessible
+ */
+#if defined LWIP_DHCP_GET_NTP_SRV && (SNTP_MAX_SERVERS > 1)
+    sntp_setservername(1, "pool.ntp.org");
+#else
     sntp_setservername(0, "pool.ntp.org");
+#endif
+
     sntp_set_time_sync_notification_cb(time_sync_notification_cb);
 #ifdef CONFIG_SNTP_TIME_SYNC_METHOD_SMOOTH
     sntp_set_sync_mode(SNTP_SYNC_MODE_SMOOTH);
 #endif
     sntp_init();
+
+    ESP_LOGI(TAG, "List of configured NTP servers:");
+
+    for (uint8_t i = 0; i < SNTP_MAX_SERVERS; ++i){
+        if (sntp_getservername(i)){
+            ESP_LOGI(TAG, "server %d: %s", i, sntp_getservername(i));
+        } else {
+            const ip_addr_t * _ip = sntp_getserver(i);
```

Comment:
Nitpick: variables starting with underscore are reserved for compiler use
```suggestion
            const ip_addr_t * ip = sntp_getserver(i);
```